components/molecules/RepoCardProfile/repo-card-profile.tsx

```diff
@@ -17,7 +17,7 @@ const RepoCardProfile = ({ avatar, orgName, repoName, prCount, issueCount, class
       <div className="h-max group-hover:border rounded-lg">
         <Avatar initials="BD" size={40} avatarURL={avatar} />
       </div>
-      <div className="flex flex-col gap-1 h-[46px]">
+      <div className="flex flex-col gap-1 sm:h-[60px]">
```

Comment:
This is not correct either. You are now removing the height and still adjusting the pixels. I appreciate the attempt but this is now removing height css from desktop screens all together and only applying mobile.

Reply:
could you use `h-16` -> `64px` ???

Reply:
Visually this looks great now. This comment needs to be addressed. https://github.com/open-sauced/insights/issues/1522